src/Event.vala

```diff
@@ -48,13 +49,13 @@ public class DateTime.Event : GLib.Object {
     public string get_label () {
         var summary = component.get_summary ();
         if (day_event) {
-            return summary;
+            return "%s\n%s".printf (summary, _("All Day"));
         } else if (alarm) {
-            return "%s - %s".printf (start_time.format (Util.TimeFormat ()), summary);
+            return "%s %s\n%s".printf (_("Alarm:"), start_time.format (Util.TimeFormat ()), summary);
```

Comment:
Please don't put newlines in the middle of strings. Use two labels so we can lay them out accurately